src/SprykerEco/Zed/Unzer/UnzerConstants.php

```diff
@@ -12,18 +12,98 @@ interface UnzerConstants
     /**
      * @var string
      */
-    public const UNZER_PAYMENT_METHOD_CARD = 'card';
+    public const UNZER_PAYMENT_METHOD_ALIPAY = 'alipay';
```

Comment:
Looks like a part of [module config](https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-ModuleConfig) not [constants](https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-Constants).
I mean all const here.

Reply:
Why not? In config we usually keep some configurable things - e.g. the array of available payment methods.
It was made in such way as for best SprykerEco practices (:

Reply:
Will not be changed as discussed